### PR TITLE
[PR] Move youtube-embed.js file and include in Grunt processing

### PR DESCRIPTION
**Pull request by philcable**
Nothing urgent, it just occurred to me that I stuck this file in the wrong place when I first included it. This moves it and makes adjustments to meet coding standards.

### CODE REVIEW

**Comment by philcable** [comment]
@jeremyfelt #reviewmerge?

File: src/js/youtube-embed.js [hunk 1]
```diff
@@ -20,32 +20,33 @@
...
-			player = new YT.Player( 'youtube-video-' + video_id, {
+			player = new window.YT.Player( "youtube-video-" + video_id, {
 				height: video_height,
 				width: video_width,
 				videoId: video_id,
 				playerVars: {
 					modestbranding: 1,
 					showinfo: 0,
-					controls: 0,
+					controls: 1,
```
Comment (jeremyfelt) [inline src/js/youtube-embed.js:36]: Is this shift from `0` to `1` intentional here?

**Review by jeremyfelt — APPROVED** [review]
:+1: looks good. I left one note, so not merging yet. If that's okay the way it is, then good to go for merge.

File: src/js/youtube-embed.js (hunk 1 above)
Comment (philcable) [reply src/js/youtube-embed.js:36]: It is indeed, but I confess that it was an executive decision I made after noticing that some of the videos on the site were 10+ minutes long. Let me double check with Charles to make sure he's cool with it.

Comment (philcable) [reply src/js/youtube-embed.js:36]: @jeremyfelt @charlesjdillon gave the 👍  on adding controls!

**Pull request merged**